vector/src/main/scala/geotrellis/vector/Extent.scala

```diff
@@ -43,9 +43,9 @@ object ProjectedExtent {
  */
 case class Extent(xmin: Double, ymin: Double, xmax: Double, ymax: Double) {
 
-  // Validation: Do not accept extents min values greater than max values.
-  if (xmin > xmax) { throw ExtentRangeError(s"Invalid Extent: xmin must be less than xmax (xmin=$xmin, xmax=$xmax)")  }
-  if (ymin > ymax) { throw ExtentRangeError(s"Invalid Extent: ymin must be less than ymax (ymin=$ymin, ymax=$ymax)") }
+  // Validation: Do not accept extents min values greater than or equal to max values (allows empty extents).
+  if (xmin >= xmax) { throw ExtentRangeError(s"Invalid Extent: xmin must be less than or equal to xmax (xmin=$xmin, xmax=$xmax)")  }
```

Comment:
@lossyrob Want to note, that `xmin >= xmax` *not* allowing empty extents; the previous condition `xmin > xmax` accepted: other words all extents with `xmin <= xmax` were accepted; should I rollback changes?